src/util/intOps.ml

```diff
@@ -14,8 +14,8 @@ sig
   (* Constants *)
   val zero : t
   val one : t
-  val lower_bound : t option
-  val upper_bound : t option
+  val lower_bound : t option (* TODO: unused *)
+  val upper_bound : t option (* TODO: unused *)
```

Comment:
Those seem a bit strange here. IIRC, `IntOpsBase` is the module type we use for single values, not modules, ....
Maybe @jerhard knows what they are intended for?

Reply:
I suppose at some point (maybe pre-bigints) it was necessary to know the `min_int` and `max_int` for these different types.